docs/apps/components/alert.md

```diff
@@ -0,0 +1,59 @@
+# Alert
+
+## Description
+
+The Alert component can be used to display success, warning, information or error-type boxes.
```

Comment:
Explanation would be better on how to add the Alert Component starting from 
1) Creating New App,
2) Selecting the App
3) Creating Page Builder for the App.

(Ignore if already the procedure exists, but a redirection link would be better to explain the procedure)
Below is what I encountered to create an Alert Component.
1) Create App with a name (Alert-App-Ramu)
2) We need to select 'Alert-App-Ramu' from the dropdown above the folder structure.
3) After selecting the app it shows a plus (right side) which is a drop down list, select Page Builder
4) Create a page builder - (Alert-Page-Ramu).
5) After the page builder is created, select it from folder structure (Alert-Page-Ramu).
6) Once loaded, it shows with page builder sections where you can alert component from the left navigation (Layout).

Reply:
!!! Note
    [Click here](https://bani-appsection--connexcs-docs.netlify.app/apps/page-builder/#steps-to-use-components-in-the-page-builder) for the detailed steps on how to use the components.
    
    Please check the steps:
https://bani-appsection--connexcs-docs.netlify.app/apps/page-builder/#steps-to-use-components-in-the-page-builder

Reply:
1. ## Steps to use the Components

[Click here](https://bani-appsection--connexcs-docs.netlify.app/apps/page-builder/#steps-to-use-components-in-the-page-builder target="_blank") for the detailed steps on how to use the components.